Woof/WoofTests/ViewModel/LoginViewModelTests/LoginViewModelSetUserRoleTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class LoginViewModelSetUserRoleTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.testPreferencesStorage
+    private let testLoginViewModel = LoginViewModel()
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+
+    func testExistsInAPI() {
+        _ = testLoginViewModel.set(.sitter)
+    }
+
+    func testReturnsTrueWhenPassAllPossibleRoles() {
+        Role.allCases.forEach {
+            XCTAssertTrue(testLoginViewModel.set($0))
+        }
+    }
+
+    func testOverridesThePreviouslySavedRole() {
```

Comment:
nit:
```suggestion

    func testReturnsExpectedValueWhenOverridesThePreviouslySavedRole() {
```

Reply:
I removed the return for this method and rewrote this test:
``` swift
func testReturnsTrueWhenSetEveryPossibleRole() {
        Role.allCases.forEach {
            XCTAssertTrue(testLoginViewModel.set($0))
        }
 }
 ```